R/load_pop_data.R

```diff
@@ -107,54 +104,51 @@ set_age <- function(object, ...){
 }
 
 #' @export
-set_age.pop_data <- function(object, age = 'Age', standardize, ...){
+set_age.pop_data <- function(object, age = 'Age', standardize = TRUE, ...){
 
-  if(standardize)
-  {
-    # set age attribute
-    attr(object, "age_var") <- 'age'
-
-    # rename provided column
-    if(age %in% colnames(object))
-    {
-      object <- object %>%
-        rename('age' = age)
-    } else
-    {
-      cli::cli_abort(
-        '`age = "{age}"` is not a valid input;
-      the age column in not available in the data set')
-    }
-
-
```

Comment:
we want to find the right column first, and possibly standardize second; I moved this code to later in this function.